meta/src/parser.rs

```diff
@@ -990,16 +1086,7 @@ mod tests {
             parser: PestParser,
             input: "a = { b ~ }",
             rule: Rule::grammar_rules,
-            positives: vec![
-                Rule::opening_paren,
-                Rule::positive_predicate_operator,
-                Rule::negative_predicate_operator,
-                Rule::_push,
-                Rule::identifier,
-                Rule::insensitive_string,
-                Rule::quote,
-                Rule::single_quote
-            ],
+            positives: vec![Rule::term],
```

Comment:
no idea why I had to change this. is this a bug on master or did my added rule really change something?

Reply:
I've fixed it in #343.